src/views/Header.js

```diff
@@ -71,6 +76,24 @@ class Header extends React.PureComponent<void, HeaderProps, HeaderState> {
     return this._getHeaderTitleString(lastScene);
   }
 
+  _getBackButtonTitleStyle(scene: NavigationScene): ?Style {
+    const lastScene = this._getLastScene(scene);
+    if (!lastScene) {
+      return null;
+    }
+    const {
+      headerBackTitleStyle,
```

Comment:
I believe that `headerBackTitleStyle` and `headerTitleStyle` you use for styling back button should be taken from a "current" scene, not the previous one. I think as far as styling is concerned, it's responsibility of currently displayed scene to define it.

CC: @satya164